docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   redis:
     image: redis:3.0
   app:
-    build: .
+    image: voxmedia/docker_base_images:ruby_2.2-0.1
```

Comment:
Does this image have ruby 2.2.0 version? Autotune runs on 2.2.2 https://github.com/voxmedia/vox-autotune/blob/master/.ruby-version